src/index.js

```diff
@@ -58,6 +58,8 @@ module.exports = (() => {
 
         if (plugin) {
           plugin(editor, config.pluginsOpts[pluginId] || {});
+        } else if (typeof pluginId === 'function') {
+          pluginId(editor, config.pluginsOpts[pluginId] || {});
```

Comment:
In this case, you can't pass options, so `config.pluginsOpts[pluginId]` doesn't make sense.
You could pass them only in this way
```js
plugins: [
	myPluginNoOptions,
	e => myPluginWithOptions(e, { ... }),
]
```

Reply:
I was on the fence about this too.

- On one hand you can use objects as keys (see: https://jsfiddle.net/n0ou7x84/)
- On the other hand, this is a bit of an obscure syntax for most people

I left it in because I didn't imagine that it would cause much harm, the options may be null or ignored in many cases.

In terms of docs, though, I think the option-passing approach you explained is the more user-friendly approach and should be included as the recommended way of passing in plugin options.